packages/suite/src/components/suite/modals/TransactionDetail/components/AmountDetails/index.tsx

```diff
@@ -218,23 +220,25 @@ const AmountDetails = ({ tx, txDetails, isTestnet }: Props) => {
                     fourthColumn={showFiat && <FiatValue amount={amount} symbol={assetSymbol} />}
                     color="light"
                 />
-                {/* TX FEE */}
-                <AmountRow
-                    firstColumn={<Translation id="TR_TX_FEE" />}
-                    secondColumn={<FormattedCryptoAmount value={tx.fee} symbol={tx.symbol} />}
-                    thirdColumn={
-                        showHistoricalRates && (
-                            <FiatValue
-                                amount={tx.fee}
-                                symbol={tx.symbol}
-                                source={tx.rates}
-                                useCustomSource
-                            />
-                        )
-                    }
-                    fourthColumn={showFiat && <FiatValue amount={tx.fee} symbol={tx.symbol} />}
-                    color="light"
-                />
+                {/* TX FEE (hide field for pending eth txs as we don't know the exact amount til tx is mined) */}
```

Comment:
https://github.com/trezor/trezor-suite/issues/3374#issuecomment-776749124

Doesn't make more sense to hide this field since we can't know the exact fee spent till the transaction is mined? It seems weird to me to show worst case scenario and then after tx gets mined replace that value with real fee that will be different from the previous shown fee.

Reply:
if you are sending transaction then you've confirmed gasLimit * gasPrice fee, you want to see this value in pending detail.
even if this value is not final and may be changed it's still better than see '0' or nothing.

finally, i need this value in "replace transaction" form anyway :)